pages/common/transmission-daemon.md

```diff
@@ -0,0 +1,21 @@
+# transmission-daemon
+
+> Daemon controlled with `transmission-remote` or its web interface.
+> See also: `transmission`.
+> More information: <https://manned.org/transmission-daemon>.
+
+- Start a headless Transmission session:
+
+`transmission-daemon`
+
+- Specify a directory to watch for new torrent files:
+
+`transmission-daemon --watch-dir {{path/to/directory}}`
+
+- Generate a configuration file for the daemon:
+
+`transmission-daemon --dump-settings > {{path/to/file.json}}`
+
+- Run the daemon with explicit settings for the web interface:
+
+`transmission-daemon --auth --username {{username}} --password {{password}} --port {{9091}} --allowed "{{127.0.0.1}}"`
```

Comment:
IPs or hotnames don't need quotation marks :thinking: 
```suggestion
`transmission-daemon --auth --username {{username}} --password {{password}} --port {{9091}} --allowed {{127.0.0.1}}`
```

Reply:
Oh, I guess you're right. I borrowed this from the Arch Wiki, and I assume they put quotes since it's actually specifying a string where a list of IPs can be included. Not really necessary here though.